Source/WebCore/platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp

```diff
@@ -100,6 +108,23 @@ RefPtr<FontCustomPlatformData> createFontCustomPlatformData(SharedBuffer& buffer
     return adoptRef(new FontCustomPlatformData(fontDescriptor.get(), WTFMove(creationData)));
 }
 
+RefPtr<FontCustomPlatformData> createFontCustomPlatformDataMemorySafe(SharedBuffer& buffer, const String& itemInCollection)
```

Comment:
Why isn't this a static function on `FontCustomPlatformData` ? `FontCustomPlatformData::createMemorySafe()`

Reply:
The boring reason is that I put it in the same scope as `createFontCustomPlatformData`. I can make both static functions.

Reply:
Hm, I don't know if I should do that. `createFontCustomPlatformData` is `WEBCORE_EXPORT`. Am I going to break something/someone if I change its ABI name? If I leave it there, should I still move `createFontCustomPlatformDataMemorySafe`?

Reply:
WebCore is only used by WebKit there is no ABI compatibility issue. We own all the callers, in the same codebase.